python/pyspark/pandas/namespace.py

```diff
@@ -1629,9 +1629,28 @@ def to_datetime(
     DatetimeIndex(['1960-01-02', '1960-01-03', '1960-01-04'], dtype='datetime64[ns]', freq=None)
     """
 
+    _unit_map = {
```

Comment:
Looks fine. can you mention that this was from pandas'? feel free to point out pandas' file name together here.

Reply:
Thank you, @HyukjinKwon. So where should I mention that? In this PR or something else?

Reply:
mention that it was copied from pandas' logic, and comment the file name in pandas.